.gitignore

```diff
@@ -27,3 +27,4 @@ venv
 .venv
 
 /build
+kasa/launch.json
```

Comment:
Please remove this.

Reply:
yes. Just removed this. I'm new to git so I don't know how to eliminate just one file from the pull request vs undoing the changes.